tests/util.rs

```diff
@@ -0,0 +1,14 @@
+use std::fs::OpenOptions;
```

Comment:
I have strong dislike for the name "util" (or "misc" and in some cases "lib") because it doesn't convey information. I think `create_bincode_file.rs` would be a good name but feel free to suggest other.